composer.json

```diff
@@ -13,8 +13,8 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-config": "2.2.3",
-        "zendframework/zend-di": "2.2.3"
+        "zendframework/zend-config": "~2.3",
```

Comment:
@bryanlatten ~2.3 means >=2.3. if someone has 2.2.3 Phystrix will stop working for them, although there is no reason. Please correct me if I'm wrong